### DISCUSSION TITLE
it/blog/introducing-decap-cms-your-go-to-solution-for-content-management

### DISCUSSION THREAD

**Discussion by giscus** [majesticooss/astros#21] [Blog]
# it/blog/introducing-decap-cms-your-go-to-solution-for-content-management

Astros is a starter website template for Astro built with TailwindCSS and alpine js.

http://173.232.195.175/it/blog/introducing-decap-cms-your-go-to-solution-for-content-management

**Comment by satybaldin** [comment]
hello

**Comment by elias-michaias** [comment]
Hello!

**Comment by geroxima** [comment]
Love the site!
